src/components/tables/RolesTable.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <q-table
+    class="shadow-5"
+    table-header-class="bg-grey-3"
+    hide-bottom
+    :pagination="pagination"
+    :columns="columns"
+    :rows="roles"
+    :row-key="rowKey"
+    data-cy="roles_table"
```

Comment:
since `page` was removed from data-cy here, we could also remove it from data-cy in `libraries_table`  to be consistent?

Reply:
done